### PR TITLE
Italian translation of 1.mdx

### PR DESCRIPTION
**Pull request by CaterinaBi**

### CODE REVIEW

**Review by sharkovsky — COMMENTED** [review]
Hi @CaterinaBi, your commit contains the original english files 1.mdx, 2.mdx, etc.. in the main folder, in addition to those in the chapters/en folder. 
I suspect this is a mistake, and you probably need to remove them. To update a pull request, you only need to commit the new changes and push them to your fork: they will automatically be included in this pull request. 
Additionally, it is usually nicer to integrate the "upstream changes" (i.e. the changes in the original huggingface repository) before submitting a pull request. Doing so is not so straightforward, but it's a nice pattern to learn. You would probably need to do something like `git pull upstream main` in your forked repository, correct any merge conflicts (I don't expect there to be) and re-push your changes to your fork.

**Comment by CaterinaBi** [comment]
Hi @sharkovsky, thanks for the advice, it's my first pull request ever so you know...

I'll try to fix the problems. Don't hesitate to give me feedback! Thank you!

**Comment by CaterinaBi** [comment]
@sharkovsky, I've deleted the files in main. Now I'm not sure how to pull and merge the changes in the original huggingface repo to my fork?? It'd be great if you could give me some hints. Also, I've got another file in its final form, do I create a new pull request each and every time a file is done? Thanks!

**Comment by sharkovsky** [comment]
Ciao Caterina!

I'd love to help. Are you using the command line for git, or some other tool/GUI?

Git has this concept of `remote`: you should have two right now, `origin` which points to your forked repository, and `upstream` pointing to the original huggingface repository. 

By default you usually push and pull from origin, but if you want to integrate the latest changes from huggingface you need to pull the changes from upstream. With command line, this requires:
```bash
git pull upstream main
```

When you `git pull` it will fetch the changes from the main branch of the upstream remote repository, and try to merge them. I don't expect there to be conflicts, but if there are you can try to solve them by manually updating the files with conflicts and running 
```bash
git merge --continue
```
in extreme cases, if you are getting really confused you can always
```bash
git merge --abort
```

If you need, we can set up a google meet or some other quick call!

File: chapters/it/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
+- title: 0. Installazione
+  sections:
+  - local: chapter0/1
+    title: Introduzione
+
+- title: 1. Modelli Transformer
+  sections:
+  - local: chapter1/1
+    title: Introduzione
+  - local: chapter1/2
```
Comment (lewtun) [inline chapters/it/_toctree.yml:10]: Thanks for translating the whole ToC, but we only need to include the sections for which there are MDX files in Italian. In your case, we have `chapter0/1.mdx` and `chapter1/1.mdx` translated, so the `_toctree.yml` file should just include:

```yaml
- title: 0. Installazione
  sections:
  - local: chapter0/1
    title: Introduzione

- title: 1. Modelli Transformer
  sections:
  - local: chapter1/1
    title: Introduzione
```

**Review by lewtun — COMMENTED** [review]
Hi @CaterinaBi thank you for this great contribution and congrats on your first PR to the Hugging Face ecosystem 🔥 ! 

As @sharkovsky mentioned, we should only include the files that have been translated, so all the original English ones can be deleted. Please let us know if you need some help with Git and we'll be happy to provide it 🤗

**Comment by sharkovsky** [comment]
Hi @CaterinaBi , how are things going? Let me know if i can help in any way!

File: chapters/it/_toctree.yml (hunk 1 above)
Comment (CaterinaBi) [reply chapters/it/_toctree.yml:10]: Ok! Thanks!

**Comment by CaterinaBi** [comment]
> Hi @CaterinaBi , how are things going? Let me know if i can help in any way!

Hi @sharkovsky , I'm so sorry I ghosted you, I had all sorts of weird things happening to me: I first found myself without any network on the Italian Dolomites for 10 days (thankfully I had planned to take time off from work, so I didn't have any planned meetings!), then I came back to England and realised I forgot my sim card in my mother's phone when she brought me her Italian sim to see whether I'd be able to get some work done using that as a hotspot (that network was terrible too, by the way). So I was locked out of all websites and applications that required a double identification. I should have had GitHub security coded but couldn't find them anywhere... A modern nightmare!

Anyway, I'll be trying to get things right tonight and come back to you if I see I can't update the pull request alone. Because of all that happened I'm also so late with the translations, I feel super guilty!

**Comment by lewtun** [comment]
Welcome back @CaterinaBi 🤗 ! Please let us know if you need any help :)

**Comment by CaterinaBi** [comment]
> Welcome back @CaterinaBi 🤗 ! Please let us know if you need any help :)

Hello @lewtun! Sorry I disappeared, I feel so bad about this! I've tried to update the pull request, will you please let me know if I've done all correctly this time?

Just to double check: how do you want me to proceed? Do I open a pull request each time I'm done translating a file (and updating the ToC)?

File: chapters/it/_toctree.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- title: 0. Installazione
+  sections:
+  - local: chapter0/1
+    title: Introduzione
```
Comment (lewtun) [inline chapters/it/_toctree.yml:4]: I think we lost the piece for `chapter1/1` 😬 . Could you add the following below this line:

```yaml
- title: 1. Modelli Transformer
  sections:
  - local: chapter1/1
    title: Introduzione
```

**Review by lewtun — COMMENTED** [review]
Thanks for the fixes @CaterinaBi - this is looking great! There's just a small fix needed for the ToC and could you please add the `it` language code to the `languages` field of the `build_documentation.yml` and `build_pr_documentation.yml` files? You want to add something like:

```
languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN
```

**Comment by lewtun** [comment]
> Just to double check: how do you want me to proceed? Do I open a pull request each time I'm done translating a file (and updating the ToC)?

Yes, this would be great! From other translations, I think it's a bit faster if you group a few sections together in a single PR, but of course we'll happily accept single sections 🤗

File: chapters/it/_toctree.yml (hunk 2 above)
Comment (lewtun) [reply chapters/it/_toctree.yml:4]: Thanks for this! It seems that we just have some merge conflicts to resolve on the `build_pr_documentation.yml` and `build_documentation.yml` files - would you be happy to fix that?

Comment (CaterinaBi) [reply chapters/it/_toctree.yml:4]: Ok so I've just resolved my first conflicts ever. Hopefully I didn't make a mess!

**Review by lewtun — APPROVED** [review]
Thanks for the final fix! I'm now running the CI to check if the docs build and we should soon see the first part of the course in Italian 🇮🇹 !

**Comment by CaterinaBi** [comment]
> Thanks for the final fix! I'm now running the CI to check if the docs build and we should soon see the first part of the course in Italian 🇮🇹 !

Great! Thank you very much @lewtun, I owe you a couple of drinks!

**Comment by lewtun** [comment]
Woohoo it worked 🥳 ! Merging this now :)

Screen Shot 2022-04-26 at 23 04 29

**Comment by CaterinaBi** [comment]
Yay! Thanks again!

**Pull request merged**